tests/test_oom_logger.py

```diff
@@ -193,6 +241,21 @@ def log_line():
     )
 
 
+@pytest.fixture
+def log_line_containerd():
+    return LogLine(
+        timestamp=1720128512,
+        hostname="dev208-uswest1adevc",
+        container_id="e216d2f1e6c625d363c71edb6b3cbab5a9e1b447641b61028d0b94b077adf27c",
+        cluster="fake_cluster",
+        service="fake_service",
+        instance="fake_instance",
+        process_name="python3",
+        mesos_container_id="mesos-a04c14a6-83ea-4047-a802-92b850b1624e",
```

Comment:
same here, but 
```suggestion
        mesos_container_id="mesos-null",
```
instead of deleting this line